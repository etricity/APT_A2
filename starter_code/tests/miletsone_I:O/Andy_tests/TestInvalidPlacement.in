2
TestCase
place G3 at B2
place B6 at D0