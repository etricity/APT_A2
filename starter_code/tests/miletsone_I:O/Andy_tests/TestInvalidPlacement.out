A, it's your turn
Score for A: 12
Score for B: 16

  0  1  2  3  4  5  6  7  8  9  10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25
  -----------------------------------------------------------------------------
A |     |     |     |     |     |     |     |     |     |     |     |     |    
B    |     |  R6 |  O4 |     |     |     |     |     |     |     |     |     | 
C |  G6 |  B6 |     |  O2 |     |     |     |     |     |     |     |     |    
D    |  Y6 |  B1 |     |  O3 |     |     |     |     |     |     |     |     | 
E |  P6 |     |  B2 |  P3 |  O5 |     |     |     |     |     |     |     |    
F G6 |     |     |  B3 |     |  O1 |     |     |     |     |     |     |     | 
G |     |     |     |     |     |     |     |     |     |     |     |     |    
H    |     |     |     |     |     |     |     |     |     |     |     |     | 
I |     |     |     |     |     |     |     |     |     |     |     |     |    
J    |     |     |     |     |     |     |     |     |     |     |     |     | 
K |     |     |     |     |     |     |     |     |     |     |     |     |    
L    |     |     |     |     |     |     |     |     |     |     |     |     | 
M |     |     |     |     |     |     |     |     |     |     |     |     |    
N    |     |     |     |     |     |     |     |     |     |     |     |     | 
O |     |     |     |     |     |     |     |     |     |     |     |     |    
P    |     |     |     |     |     |     |     |     |     |     |     |     | 
Q |     |     |     |     |     |     |     |     |     |     |     |     |    
R    |     |     |     |     |     |     |     |     |     |     |     |     | 
S |     |     |     |     |     |     |     |     |     |     |     |     |    
T    |     |     |     |     |     |     |     |     |     |     |     |     | 
U |     |     |     |     |     |     |     |     |     |     |     |     |    
V    |     |     |     |     |     |     |     |     |     |     |     |     | 
W |     |     |     |     |     |     |     |     |     |     |     |     |    
X    |     |     |     |     |     |     |     |     |     |     |     |     | 
Y |     |     |     |     |     |     |     |     |     |     |     |     |    
Z    |     |     |     |     |     |     |     |     |     |     |     |     | 
  -----------------------------------------------------------------------------

Your hand is
G3,B6,Y1,O6,R5,G2

> Invalid Position!!

> B, it’s your turn
Score for A: 16
Score for B: 16

  0  1  2  3  4  5  6  7  8  9  10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25
  -----------------------------------------------------------------------------
A |     |     |     |     |     |     |     |     |     |     |     |     |    
B    |     |  R6 |  O4 |     |     |     |     |     |     |     |     |     | 
C |  G6 |  B6 |     |  O2 |     |     |     |     |     |     |     |     |    
D    |  Y6 |  B1 |     |  O3 |     |     |     |     |     |     |     |     | 
E |  P6 |     |  B2 |  P3 |  O5 |     |     |     |     |     |     |     |    
F G6 |     |     |  B3 |     |  O1 |     |     |     |     |     |     |     | 
G |     |     |  G3 |     |     |     |     |     |     |     |     |     |    
H    |     |     |     |     |     |     |     |     |     |     |     |     | 
I |     |     |     |     |     |     |     |     |     |     |     |     |    
J    |     |     |     |     |     |     |     |     |     |     |     |     | 
K |     |     |     |     |     |     |     |     |     |     |     |     |    
L    |     |     |     |     |     |     |     |     |     |     |     |     | 
M |     |     |     |     |     |     |     |     |     |     |     |     |    
N    |     |     |     |     |     |     |     |     |     |     |     |     | 
O |     |     |     |     |     |     |     |     |     |     |     |     |    
P    |     |     |     |     |     |     |     |     |     |     |     |     | 
Q |     |     |     |     |     |     |     |     |     |     |     |     |    
R    |     |     |     |     |     |     |     |     |     |     |     |     | 
S |     |     |     |     |     |     |     |     |     |     |     |     |    
T    |     |     |     |     |     |     |     |     |     |     |     |     | 
U |     |     |     |     |     |     |     |     |     |     |     |     |    
V    |     |     |     |     |     |     |     |     |     |     |     |     | 
W |     |     |     |     |     |     |     |     |     |     |     |     |    
X    |     |     |     |     |     |     |     |     |     |     |     |     | 
Y |     |     |     |     |     |     |     |     |     |     |     |     |    
Z    |     |     |     |     |     |     |     |     |     |     |     |     | 
  -----------------------------------------------------------------------------

Your hand is
R3,O2,G4,G2,Y3,P1

>